gpt_engineer/main.py

```diff
@@ -26,6 +41,9 @@ def main(
 ):
     logging.basicConfig(level=logging.DEBUG if verbose else logging.INFO)
 
+    # Call the new function before any actions that require environment variables
```

Comment:
Coding convention is to only add comments for "complex" pieces of code – this function name is so good it's self explanatory

```suggestion
```